includes/class-wc-payments-apple-pay-registration.php

```diff
@@ -313,9 +313,9 @@ public function register_domain_with_apple() {
 	 * Process the Apple Pay domain verification if proper settings are configured.
 	 */
 	public function verify_domain_if_configured() {
-		// If Payment Request Buttons are not enabled, or account is not live,
+		// If Payment Request Buttons are not enabled,
```

Comment:
Is this method used for PRB and ECE? If not maybe we change PRB to "Express Checkout Elements"?